src/core/components/avatars/avatar/avatar.tsx

```diff
@@ -33,13 +33,27 @@ const responsiveSizeClasses: ResponsiveAttributeClassMap<AvatarSize> = {
         '2xl': '2xl:w-6 2xl:h-6',
     },
     md: {
+        sm: 'w-8 h-8',
+        md: 'md:w-8 md:h-8',
+        lg: 'lg:w-8 lg:h-8',
+        xl: 'xl:w-8 xl:h-8',
+        '2xl': '2xl:w-8 2xl:h-8',
+    },
+    lg: {
```

Comment:
this is kind n
nitpicky, but do you think we should do something like this:

```
sm: size-8
md: md:size-8
lg: lg:size-8
... etc
```

If so we could update in a number of place where use a circular shape like Spinner as well.